src/modules/form/components/viewable/DynamicView.tsx

```diff
@@ -53,19 +58,13 @@ const DynamicView: React.FC<
           disabledDependencyMap,
         }}
       >
-        <Grid
-          container
-          direction='column'
-          spacing={2}
-          data-testid='dynamicView'
-          {...GridProps}
-        >
+        <DynamicFormLayout variant={variant} GridProps={GridProps}>
```

Comment:
This removes `data-testid='dynamicView'`. Based on grepping both frontend and backend codebases, I _think_ that is currently unused (e.g. by system tests), but let me know if there's something I'm not thinking of here.